core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -1753,9 +1761,9 @@ class ConnectionQuotas(config: KafkaConfig, time: Time, metrics: Metrics) extend
      * The average is out of all throttle times > 0, which is consistent with the bandwidth and request quota throttle
```

Comment:
need to update comment for this method, because the beginning of the comment says that throttle is `due to hitting connection rate quota.`